[docs] Fix npx folder confusion (#2346)
mui/mui-toolpad: docs/data/toolpad/getting-started/installation.md

## Create a Toolpad app

Start by opening a terminal and `cd` into a folder you'd like to create your applicarion in. Then run the command
Start by opening a terminal and `cd` into a folder you'd like to create your applicarion in.
Then run the command:

```bash
npx create-toolpad-app my-app
<codeblock storageKey="package-manager">

```bash npm
npx create-toolpad-app@latest
```

```bash yarn
yarn create toolpad-app
```

> > This will run the `create-toolpad-app` CLI which initializes the directory `./my-app` with a Toolpad application.

Author: This is for quotes

</codeblock>

:::info

> This will run the `create-toolpad-app` CLI which initializes a directory (`./my-toolpad-app` by default) with a Toolpad application.

Reviewer: I believe it will ask for a folder, @bharatkashyap ?

Author: It asks yes.

Reviewer 2: ```suggestion
This will run the `create-toolpad-app` CLI which asks for the folder in which you wish to initialise your Toolpad application.
```
